scripts/prepare.sh

```diff
@@ -44,3 +44,12 @@ mkdir -p "$core_assets"
 # Bundle currency, swap, & rate plugins:
 echo Webpacking plugins...
 node ./node_modules/.bin/webpack
+
+# Env file setup:
+brew install jq
```

Comment:
This would break Horribly on every system that is not a mac, which is ALL the build systems.

All the automated CI/CD tools (Travis, Jenkins) and even a big portion of our developers uses linux and even god forbid Windows.